### DISCUSSION TITLE
오픈채팅방 공지쪽 댓글부분은 어디에서 찾을 수 있나요?

### DISCUSSION THREAD

**Discussion by starha0128** [storycraft/node-kakao#385] [Q&A]
setNoticeMeta와 관련된 것으로 보이는데.
정확히 어디에 있는지를 모르겠네요.

**Comment by ghost** [comment]
현재 V4에는 오픈채팅방 글 관련 API가 구현된게 없습니다. V3을 참고해보세요.

**Reply by starha0128** [reply]
V3에서 해당 부분이 어디 있는거죠? 계속 보고 있는데 찾기가 힘드네요.

**Comment by generalxd** [comment]
공지쪽 댓글로 뭐하시려구요
방장말고는 아무권한없던데

**Reply by starha0128** [reply]
부방으로 해놓고 광고를 공지로 등록해버리거나 공지쪽 댓글에도 광고를 달길래 공지를 삭제해버리고 새 공지를 달던가.
아니면 댓글만 삭제해버리려고요.

**Reply by generalxd** [reply]
> 부방으로 해놓고 광고를 공지로 등록해버리거나 공지쪽 댓글에도 광고를 달길래 공지를 삭제해버리고 새 공지를 달던가.
> 아니면 댓글만 삭제해버리려고요.

부방장은 댓글삭제권한이없더라구요 ㅎㅎ 글삭제도 당연히안되구요

**Reply by starha0128** [reply]
그렇군요 ㅠㅠ

**Reply by starha0128** [reply]
그러면 방장으로 설정하면은 댓글삭제나 공지 삭제 기능을 찾으면 사용할 수 있다는 말씀이시죠? 빨리 찾아봐야겠네요

[a comment by an automated account is not shown]
